measures/forms.py

```
@@ -177,9 +176,11 @@ def save(self, commit=True):
 
         sid = instance.sid
 
+        duty_key = f"instance_duty_sentence_{self.instance.sid}"
+        session = self.request.session
         if (
-            self.request.session[f"instance_duty_sentence_{self.instance.sid}"]
-            != self.cleaned_data["duty_sentence"]
+            duty_key in session.keys()
+            and session[duty_key] != self.cleaned_data["duty_sentence"]
```

If the measure form isn't loaded `with_duty_sentence` then when the user submits a modified duty sentence, will this just silently ignore it? 

I think the previous behaviour of throwing an error when the form was loaded incorrectly was right… can we fix the fact that objects being passed to the form don't have a duty sentence attached???